Duplicati/Library/AutoUpdater/UpdaterManager.cs

```diff
@@ -77,11 +77,17 @@ public static class UpdaterManager
         private const string RUN_UPDATED_FOLDER_PATH = "AUTOUPDATER_LOAD_UPDATE";
         private const string SLEEP_ENVNAME_TEMPLATE = "AUTOUPDATER_{0}_SLEEP";
         internal const string UPDATE_STRATEGY_ENVNAME_TEMPLATE = "AUTOUPDATER_{0}_POLICY";
+        internal const string UPDATE_NOTIFY_ENVNAME_TEMPLATE = "AUTOUPDATER_{0}_NOTIFY";
         private const string UPDATE_MANIFEST_FILENAME = "autoupdate.manifest";
         private const string README_FILE = "README.txt";
         private const string INSTALL_FILE = "installation.txt";
         private const string CURRENT_FILE = "current";
 
+        /// <summary>
+        /// Show UI notification when update is found - default is True
+        /// </summary>
+        public static bool ShowUpdateNotification { get { return Duplicati.Library.Utility.Utility.ParseBool(System.Environment.GetEnvironmentVariable(string.Format(UPDATE_NOTIFY_ENVNAME_TEMPLATE, APPNAME)), true); } }

```

Comment:
This line is quite long.  Can we remove the redundant `Duplicati.Library` and `System` qualifiers?

We can also simplify this by using the expression body syntax, but perhaps it's best to remain consistent with the rest of the file (we can refactor this separately).

Reply:
Ok will submit an update tomorrow. Thanks

Reply:
Ok done.  I agree about expression body syntax - I prefer it but I try to match the existing coding style.

I could also make long lines multiline if desired. I'm not usually a big fan of that style but can do it if desired.